### PR TITLE
start: Replace `dvc run` references in data-pipelines

### PR DESCRIPTION
**Pull request by iesahin**
This replaces `dvc run` examples in `data-pipelines`. 

Related #3309

### CODE REVIEW

**Review by shcheklein — APPROVED** [review]
`dvc stage add` doesn't actually run it, right? We should check though that we don't have anywhere text that assumes that artifacts are already created, etc. We would need to run `dvc exp run` for that.

File: content/docs/start/data-pipelines.md [hunk 1]
```diff
@@ -64,11 +64,11 @@ Please also add or commit the source code directory with Git at this point.
...
-          -o data/prepared \
-          python src/prepare.py data/data.xml
+$ dvc stage add -n prepare \
+                -p prepare.seed,prepare.split \
+                -d src/prepare.py -d data/data.xml \
+                -o data/prepared \
+                python src/prepare.py data/data.xml
 ```
 
 A `dvc.yaml` file is generated. It includes information about the command we ran
```
Comment (jorgeorpinel) [inline content/docs/start/data-pipelines.md:74]: > about the command we ran

Nothing ran though.

Comment (jorgeorpinel) [reply content/docs/start/data-pipelines.md:74]: > We would need to run dvc exp run for that.

Why not `repro` here @shcheklein ? Experiments complicates things under-the-hood and we don't want to go into those details in this page.

**Review by jorgeorpinel — CHANGES_REQUESTED** [review]
☝🏼

**Comment by jorgeorpinel** [comment]
> Related https://github.com/iterative/dvc.org/issues/3309

It doesn't close it?

File: content/docs/start/data-pipelines.md (hunk 1 above)
Comment (iesahin) [reply content/docs/start/data-pipelines.md]: Fixed that. 

`repro` is done after more commands added. I also don't think we should mention `exp run` here.

**Review by jorgeorpinel — APPROVED** [review]

File: content/docs/start/data-pipelines.md [hunk 2]
```diff
@@ -307,8 +306,8 @@ stages:
 
 </details>
 
-DVC pipelines (`dvc.yaml` file, `dvc run`, and `dvc repro` commands) solve a few
-important problems:
+DVC pipelines (`dvc.yaml` file, `dvc stage run`, and `dvc repro` commands) solve
```
Comment (jorgeorpinel) [inline content/docs/start/data-pipelines.md:309]: Typo, I think (committing)
```suggestion
DVC pipelines (`dvc.yaml` file, `dvc stage add`, and `dvc repro` commands) solve
```

**Pull request merged**